### PR TITLE
Issue31 fixing tests

### PR DESCRIPTION
**Pull request by nelsonsar**
I just reopened that... I think I removed the branch! Sorry about that!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .onion
 vendor
 tests/tmp
+.idea/
```
Comment (c9s) [inline .gitignore:4]: You can write your idea in doc/plans (and expand it as a plan later)  ;-)

**Comment by c9s** [comment]
Everything looks good! awesome! Thanks for this patch!

File: .gitignore (hunk 1 above)
Comment (nelsonsar) [inline .gitignore:4]: This is a folder of project configuration created by PhpStorm so the next developer do not need to have it :-)

Comment (c9s) [inline .gitignore:4]: Cool~ thanks ;-)

2012/9/17 Nelson Senna do Amaral notifications@github.com

> In .gitignore:
> 
> > @@ -1,3 +1,4 @@
> >  .onion
> >  vendor
> >  tests/tmp
> > +.idea/
> 
> This is a folder of project configuration created by PhpStorm so the next
> developer do not need to have it :-)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/c9s/Onion/pull/35/files#r1614843.

## 

Best Regards,

Yo-An Lin

**Pull request merged**